sequence_processing_pipeline/tests/data/good-mapping-file.txt

```diff
@@ -162,17 +162,15 @@ Sample_Name	BARCODE	center_name	center_project_name	experiment_design_descriptio
 6.22.22.RK.RH	GAAGAGGGTTGA	UCSDMI	Knight_ABTX	Samples from ABTX	NA	NA	Illumina MiSeq	Illumina EMP protocol 515fbc, 806r amplification of 16S rRNA V4	GT	1331807	6.22.22.RK.RH	FWD:GTGYCAGCMGCCGCGGTAA; REV:GGACTACNVGGGTWTCTAAT	Illumina	LNH	GTGTGYCAGCMGCCGCGGTAA	122822	1	Echo 550	ABTX_20230208_ABTX_11052	14339	UCSDMI	2/7/23	ABTX_Plates_238_239_240_242_S1_L001	230207_M05314_0346_000000000-KVMGL	ABTX_20230208_11052_Plate_238	Sequencing by synthesis	16S rRNA	V4	108379Z	NA	NA	1317793	ABTX_20230208_11052_Plate_238_6.22.22.RK.RH_F7	F7
 6.23.22.RK.FH	AATGGTTCAGCA	UCSDMI	Knight_ABTX	Samples from ABTX	NA	NA	Illumina MiSeq	Illumina EMP protocol 515fbc, 806r amplification of 16S rRNA V4	GT	1331807	6.23.22.RK.FH	FWD:GTGYCAGCMGCCGCGGTAA; REV:GGACTACNVGGGTWTCTAAT	Illumina	LNH	GTGTGYCAGCMGCCGCGGTAA	122822	1	Echo 550	ABTX_20230208_ABTX_11052	14339	UCSDMI	2/7/23	ABTX_Plates_238_239_240_242_S1_L001	230207_M05314_0346_000000000-KVMGL	ABTX_20230208_11052_Plate_238	Sequencing by synthesis	16S rRNA	V4	108379Z	NA	NA	1317793	ABTX_20230208_11052_Plate_238_6.23.22.RK.FH_H8	H8
-6.23.22.RK.LH.A	TAACAATAATTC	UCSDMI	Knight_ABTX	Samples from ABTX	NA	NA	Illumina MiSeq	Illumina EMP protocol 515fbc, 806r amplification of 16S rRNA V4	GT	1331807	6.23.22.RK.LH	FWD:GTGYCAGCMGCCGCGGTAA; REV:GGACTACNVGGGTWTCTAAT	Illumina	LNH	GTGTGYCAGCMGCCGCGGTAA	122822	1	Echo 550	ABTX_20230208_ABTX_11052	14339	UCSDMI	2/7/23	ABTX_Plates_238_239_240_242_S1_L001	230207_M05314_0346_000000000-KVMGL	ABTX_20230208_11052_Plate_238	Sequencing by synthesis	16S rRNA	V4	108379Z	NA	NA	1317793	ABTX_20230208_11052_Plate_238_6.23.22.RK.LH_G8	G8
-6.23.22.RK.LH.B	GCTCGAAGATTC	UCSDMI	Knight_ABTX	Samples from ABTX	NA	NA	Illumina MiSeq	Illumina EMP protocol 515fbc, 806r amplification of 16S rRNA V4	GT	1331807	6.23.22.RK.LH	FWD:GTGYCAGCMGCCGCGGTAA; REV:GGACTACNVGGGTWTCTAAT	Illumina	LNH	GTGTGYCAGCMGCCGCGGTAA	122822	1	Echo 550	ABTX_20230208_ABTX_11052	14339	UCSDMI	2/7/23	ABTX_Plates_238_239_240_242_S1_L001	230207_M05314_0346_000000000-KVMGL	ABTX_20230208_11052_Plate_238	Sequencing by synthesis	16S rRNA	V4	108379Z	NA	NA	1317793	ABTX_20230208_11052_Plate_238_6.23.22.RK.LH_G10	G10
+6.23.22.RK.LH	TAACAATAATTC	UCSDMI	Knight_ABTX	Samples from ABTX	NA	NA	Illumina MiSeq	Illumina EMP protocol 515fbc, 806r amplification of 16S rRNA V4	GT	1331807	6.23.22.RK.LH	FWD:GTGYCAGCMGCCGCGGTAA; REV:GGACTACNVGGGTWTCTAAT	Illumina	LNH	GTGTGYCAGCMGCCGCGGTAA	122822	1	Echo 550	ABTX_20230208_ABTX_11052	14339	UCSDMI	2/7/23	ABTX_Plates_238_239_240_242_S1_L001	230207_M05314_0346_000000000-KVMGL	ABTX_20230208_11052_Plate_238	Sequencing by synthesis	16S rRNA	V4	108379Z	NA	NA	1317793	ABTX_20230208_11052_Plate_238_6.23.22.RK.LH_G8	G8
```

Comment:
Are the deleted lines no longer good?

Reply:
I restored the removed lines and fixed them so that their orig_name values match their sample_name values, which for a non-replicate mapping file is what it should be.